INSTALL

```diff
@@ -1,47 +1,58 @@
 To get started:
 
-1. Download nginx source.
-2. Download this module.
-3. From nginx directory, run:
+1.  Download nginx source.
+2.  Download this module.
+3.  From nginx directory, run:
 
 	./configure --add-module=/path/to/module/dir
 	make
 	make install (will land in "/usr/local/nginx")
 
-4. Make a location configuration something like this:
+4.  Make a location configuration something like this:
 
-	error_log	logs/error.log	info; # Optional, but useful to see some messages
+        error_log	logs/error.log	info; # Optional, but useful to see some messages
 
-	* * * * *
+        * * * * *
 
-	location /test{
-		RavenActive on;
-		RavenLogin https://demo.raven.cam.ac.uk/auth/authenticate.html;
-		RavenLogin https://demo.raven.cam.ac.uk/auth/logout.html;
-		RavenSecretKey	qwertyui;
-		RavenLazyClock	on;
-		RavenAllow	test0001;
-		RavenDeny	test0002;
-		RavenAllow	test0003;
-		RavenDeny	test0004;
-		RavenAllow	test0005;
-	}
+        location /test {
+                RavenActive on;
+                RavenLogin https://demo.raven.cam.ac.uk/auth/authenticate.html;
+                #RavenLogin https://demo.raven.cam.ac.uk/auth/logout.html;
```

Comment:
I think that was actually meant to be RavenLogout

Reply:
That explains it (nginx wouldn't allow it to appear twice). As it happens, logout seems to work fine even without the RavenLogout line, but maybe it's failing to do something important that I missed.